src/store/components/AdminCollectionPage/index.tsx

```diff
@@ -1,59 +1,57 @@
-import { Formik } from 'formik';
 import React from 'react';
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+
+import Config from '../../../config';
+import { history } from '../../../redux_store';
 import { PublicMerchCollection } from '../../../types';
+import { fetchService } from '../../../utils';
+
 import StoreButton from '../StoreButton';
 import StoreColorInput from '../StoreColorInput';
 import StoreHeader from '../StoreHeader';
 import StoreTextInput from '../StoreTextInput';
-import Config from '../../../config';
-import { fetchService, notify } from '../../../utils';
 
 import './style.less';
-import { history } from '../../../redux_store';
 
 interface AdminCollectionPageProps {
   collection?: PublicMerchCollection | undefined;
 }
 
 interface AdminCollectionPageForm {
-  title?: string;
-  themeColorHex?: string;
-  description?: string;
+  title: string;
+  themeColorHex: string;
+  description: string;
 }
+
+const AdminCollectionPageFormSchema = Yup.object().shape({
+  title: Yup.string().min(2, 'Too Short').max(50, 'Too Long').required('Required'),
+  themeColorHex: Yup.string()
+    .matches(/^#[0-9a-fA-F]{6}$/, 'Must be a Hex Code')
+    .required('Required'),
+  description: Yup.string().min(2, 'Too Short').max(1000, 'Too Long').required('Required'),
+});
+
 const AdminCollectionPage: React.FC<AdminCollectionPageProps> = (props) => {
   const { collection } = props;
   const creatingCollection = !collection;
   const title = creatingCollection ? 'Create Collection' : 'Edit Collection';
 
+  const initialValues: AdminCollectionPageForm = {
+    title: collection?.title ?? '',
+    themeColorHex: collection?.themeColorHex ?? '',
+    description: collection?.description ?? '',
+  };
```

Comment:
`initialValues` was moved outside of the `<Formik>` tag so typing could be applied.